README.md

```diff
@@ -31,3 +31,51 @@ class AppKernel
 
 #Usages
 
+##1. Method side
+
+You can create injectable methods via dependency injections. You can apply the tag `knp_rad_prototype.prototype_method`.
+For example, if I want to expose the method `getRepository` form the `@@doctrine` service, I just have to declare a new method:
```

Comment:
Why not `@@@@@@doctrine` ? ![troll](http://static1.squarespace.com/static/507dba43c4aabcfd2216a447/t/5244609de4b0d08691bdc2ee/1380212895097/Troll+Face.jpg)